tests/testthat/test-validate.R

```diff
@@ -9,7 +9,8 @@ test_that("Error is thrown if an input value is out of bounds", {
     div_netprofit_prop_coef = 1,
     shock_year = 2030,
     term = 4,
-    company_exclusion = TRUE
+    company_exclusion = TRUE,
+    asset_type = "loans"
   ), "risk_free_rate")
 })
```

Comment:
It's generally more useful and maintainable to test internal low level functions (like `validate_*()`) through the top level funcition, e.g. `run_stress_test()` -- not directly.

Reply:
I will not do it here, since we have no integration tests for the top level function yet (would require synthetic data), but can u elaborate on this? I tend to prefer to test those isolated low level functions directly
as 

- the generation of testdata for needed arguments is closer to the code and less complex
- in case of longer running top level functions it may save time
But I assume there are good arguments agaisnt this? :)